### PR TITLE
docs: add CDC 15 to site

### LINKED ISSUE #189: add upcoming meeting to site once scheduled and when adding previous meetings recording/notes

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Resolves #189

### CODE REVIEW

File: cips/README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Read [CIP-1](./cip-1.md) for information on the CIP process.
 | 12 |   June 5, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/149) |    [Tweet Thread](https://x.com/JoshCStein/status/1798381092493504830)    |    [Recording](https://www.youtube.com/watch?v=kV5_ANNhixI)     |
 | 13 |   July 3, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/155) |    [Tweet Thread](https://x.com/JoshCStein/status/1808531780506403102)    |    [Recording](https://youtu.be/WzPi49Ge2rA)     |
 | 14 |   July 24, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/163) |    [Tweet Thread](https://x.com/JoshCStein/status/1816142425011662993), [Notes](./notes/cdc-14.md)    |    [Recording](https://www.youtube.com/watch?v=bwlpAy_KE9U)     |
+| 15 |   August 7, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/185) |    N/A    |    N/A    |
+
```
Comment (jcstein) [inline cips/README.md:24]:
```suggestion
```

File: cips/README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Read [CIP-1](./cip-1.md) for information on the CIP process.
 | 12 |   June 5, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/149) |    [Tweet Thread](https://x.com/JoshCStein/status/1798381092493504830)    |    [Recording](https://www.youtube.com/watch?v=kV5_ANNhixI)     |
 | 13 |   July 3, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/155) |    [Tweet Thread](https://x.com/JoshCStein/status/1808531780506403102)    |    [Recording](https://youtu.be/WzPi49Ge2rA)     |
 | 14 |   July 24, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/163) |    [Tweet Thread](https://x.com/JoshCStein/status/1816142425011662993), [Notes](./notes/cdc-14.md)    |    [Recording](https://www.youtube.com/watch?v=bwlpAy_KE9U)     |
+| 15 |   August 7, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/185) |    N/A    |    N/A    |
```
Comment (rootulp) [inline cips/README.md:23]: no tweet thread or recording?

Are the notes in this repo supposed to replace the tweet threads?

**Review by rootulp — APPROVED** [review]

File: cips/README.md (hunk 2 above)
Comment (jcstein) [reply cips/README.md:23]: this is for an upcoming meeting. maybe i'll leave the placeholders there as plaintext, to be linked after the meeting

thoughts on a good way to note that `August 7, 2024` is in the future without being too verbose?

Comment (jcstein) [reply cips/README.md:23]: re: notes vs. thread, may keep doing both, but not positive. the notes could be helpful for accessibility purposes IMHO

Comment (jcstein) [inline cips/README.md:23]:
```suggestion
| 15 |   August 7, 2024 (Scheduled)   | [Agenda](https://github.com/celestiaorg/CIPs/issues/185) |    Tweet Thread, Notes    |    Recording    |
```

Comment (rootulp) [reply cips/README.md:23]: oops I see why they're N/A now. I mistakenly thought this line item was for last meeting.

Comment (rootulp) [reply cips/README.md:23]: I'm fine with what you previously had or this change.

**Pull request merged**